packages/kit/package.json

```diff
@@ -6,6 +6,14 @@
   "type": "module",
   "main": "./dist/index.mjs",
   "types": "./dist/index.d.ts",
+  "exports": {
+    ".": {
+      "types": "./dist/index.d.ts",
+      "import": "./dist/index.mjs",
+      "default": "./dist/index.mjs"
```

Comment:
We might avoid default because this causes `require` to resolve into entry with wrong format. (kit is esm-only)

Reply:
Backward compatibility. I think we can drop it for kit at least 👍🏼  (since it strictly requires ESM support)